franco/Project/org.fundacionjala.automationproject/src/main/java/org/fundacionjala/automationproject/ui/HomePage.java

```diff
@@ -0,0 +1,182 @@
+package org.fundacionjala.automationproject.ui;
+
+import java.util.List;
+import org.openqa.selenium.WebDriverException;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.Select;
+
+/**
+ * Created by Franco Aldunate on 11/15/2017.
+ */
+public class HomePage extends BasePage {
+  @FindBy(linkText = "SIGN-OFF")
+  private WebElement logOut;
+
+  @FindBy(name = "tripType")
+  private List<WebElement> tripTypeList;
+
+  @FindBy(name = "passCount")
+  private WebElement passCount;
+
+  @FindBy(name = "fromPort")
+  private WebElement origin;
+
+  @FindBy(name = "fromMonth")
+  private WebElement departureMonth;
+
+  @FindBy(name = "fromDay")
+  private WebElement departureDay;
+
+  @FindBy(name = "toPort")
+  private WebElement destination;
+
+  @FindBy(name = "toMonth")
+  private WebElement returningMonth;
+
+  @FindBy(name = "toDay")
+  private WebElement returningDay;
+
+  @FindBy(name = "servClass")
+  private List<WebElement> serviceClassList;
+
+  @FindBy(name = "airline")
+  private WebElement airline;
+
+  @FindBy(name = "findFlights")
+  private WebElement findFlight;
+
+  /**
+   * <p>This method checks if the user was redirected to the home page.</p>
+   *
+   * @return whether the title is equal to the text given or not.
+   */
+  public boolean isLogged() {
+    String title = driver.getTitle();
+    return title.equalsIgnoreCase("Find a Flight: Mercury Tours:");
+  }
+
+  /**
+   * <p>This method logs out user from session.</p>
+   *
+   * @return is a MainPage object type.
+   */
+  public MainPage logOut() {
+    logOut.click();
+    return new MainPage();
+  }
+
+  /**
+   * <p>This method sets flight type.</p>
+   *
+   * @param keyWord is the value to set as flight type.
+   */
+  public void setFlightType(String keyWord) {
+    selectElementInList(tripTypeList, keyWord);
+  }
+
+  /**
+   * <p>This method sets number of passengers.</p>
+   *
+   * @param value is the number of passengers to set.
+   */
+  public void setPassengers(String value) {
+    passCount.click();
+    Select passenger = new Select(passCount);
+    passenger.selectByValue(value);
+  }
```

Comment:
I see unnecessary the method passCount.click();
this can be refactored like this:

@FindBy(name = "passCount")
private WebElement passCount;
private Select passenger ;

public void setPassengers(String value) {
passenger = new Select(passCount);
passenger.selectByValue(value);
}

or 

public void setPassengers(String value) {
passenger = new Select(webDriver.findElement(By.id("passCount")));
passenger.selectByValue(value);
}

Reply:
I understand your point, I change it by declaring the variable of Select type outside the method and I removed the .click() code line.